Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityInputSimulationProfile.cs

```diff
@@ -87,30 +112,34 @@ public class MixedRealityInputSimulationProfile : BaseMixedRealityProfile
         [Header("Hand Simulation")]
         [SerializeField]
         [Tooltip("Enable hand simulation")]
-        private HandSimulationMode handSimulationMode = HandSimulationMode.Articulated;
-        public HandSimulationMode HandSimulationMode { get => handSimulationMode; set => handSimulationMode = value; }
+        private HandSimulationMode defaultHandSimulationMode = HandSimulationMode.Articulated;
+        public HandSimulationMode DefaultHandSimulationMode => defaultHandSimulationMode;
 
         [Header("Hand Control Settings")]
         [SerializeField]
         [Tooltip("Key to toggle persistent mode for the left hand")]
-        private KeyCode toggleLeftHandKey = KeyCode.T;
-        public KeyCode ToggleLeftHandKey => toggleLeftHandKey;
+        private KeyBinding toggleLeftHandKey = KeyBinding.FromKey(KeyCode.T);
+        public KeyBinding ToggleLeftHandKey => toggleLeftHandKey;
         [SerializeField]
         [Tooltip("Key to toggle persistent mode for the right hand")]
-        private KeyCode toggleRightHandKey = KeyCode.Y;
-        public KeyCode ToggleRightHandKey => toggleRightHandKey;
+        private KeyBinding toggleRightHandKey = KeyBinding.FromKey(KeyCode.Y);
+        public KeyBinding ToggleRightHandKey => toggleRightHandKey;
         [SerializeField]
         [Tooltip("Time after which uncontrolled hands are hidden")]
         private float handHideTimeout = 0.2f;
         public float HandHideTimeout => handHideTimeout;
         [SerializeField]
         [Tooltip("Key to manipulate the left hand")]
-        private KeyCode leftHandManipulationKey = KeyCode.LeftShift;
-        public KeyCode LeftHandManipulationKey => leftHandManipulationKey;
+        private KeyBinding leftHandManipulationKey = KeyBinding.FromKey(KeyCode.LeftShift);
+        public KeyBinding LeftHandManipulationKey => leftHandManipulationKey;
         [SerializeField]
         [Tooltip("Key to manipulate the right hand")]
-        private KeyCode rightHandManipulationKey = KeyCode.Space;
-        public KeyCode RightHandManipulationKey => rightHandManipulationKey;
+        private KeyBinding rightHandManipulationKey = KeyBinding.FromKey(KeyCode.Space);
+        public KeyBinding RightHandManipulationKey => rightHandManipulationKey;
+        [SerializeField]
+        [Tooltip("Controls how mouse look control is activated")]
```

Comment:
>Controls how mouse look control is activated [](start = 18, length = 44)

I think this tooltip was copy pasted from above (but should reference hand rotaation instead).